apps/platform/src/app/app.component.ts

```diff
@@ -24,6 +25,7 @@ export class AppComponent implements OnInit, AfterViewInit {
 
   public constructor(
     public config: ConfigService,
+    public dialog: MatDialog,
```

Comment:
doesn't seem to be used, am I missing something?

Reply:
This was a debugging artifact which I have now removed.